app/templates/main/supervisorPortal.html

```diff
@@ -157,54 +160,51 @@ <h1 class="fs-1" data-toggle="collapse">Form Search</h1>
                 <div class="form-check">
                   <input class="form-check-input" name="formType" type="checkbox" value="Labor Status Form" id="original">
                   <label class="form-check-label" for="original">
-                    Original  
+                    <span class="tooltip-right"
+                      data-tooltip="A labor contract">Original
```

Comment:
This tabulation is not necessary. That same inconsistency is repeated in lines 176, 188, and 200.

Reply:
I removed the tabulation!